experimental/packages/opentelemetry-exporter-prometheus/package.json

```diff
@@ -45,6 +45,7 @@
   },
   "devDependencies": {
     "@opentelemetry/api": "^1.0.0",
+    "@opentelemetry/api-metrics": "0.27.0",
```

Comment:
Or real dependencies

Reply:
The lib check will only complain if it is part of an exported API or inside a file which is part of the exported API.

But the `serialize` function is internal and therefore not even included in `PrometheusSerializer.d.ts`. Maybe you remember https://github.com/open-telemetry/opentelemetry-js-contrib/pull/932 where I solved a similar problem by moving the internal types into an internal file.

But as metrics-api will be present anyway because of other deps it's maybe better to move it back. This protects us also from accidentally run into a libcheck problem in future just by using a type somewhere else.

Reply:
Sounds good thanks for the explanation